Source/control.cpp

```diff
@@ -1426,7 +1426,7 @@ void DrawChr(const CelOutputBuffer &out)
 	} else {
 		maxdam += plr[myplr]._pDamageMod;
 	}
-	sprintf(chrstr, "%i-%i", mindam, maxdam);
+	sprintf(chrstr, _("%i-%i"), mindam, maxdam);

```

Comment:
Not sure about this, but I guess we will get an error report if needed.

Reply:
#: Source/control.cpp:1916

Reply:
ADD_PlrStringXY(out, 366, 121, 600, "you want to remove?", COL_GOLD);

Reply:
Thanks, i have added it.